### PR TITLE
Fix criterion 7.5

### PR DESCRIPTION
**Pull request by nicolaskempf57**
Close https://github.com/etalab/data.gouv.fr/issues/809

### CODE REVIEW

**Review by maudetes — APPROVED** [review]
Looks good to me! :muscle:

File: theme/less/components/toast.less [hunk 1]
```diff
@@ -129,5 +129,11 @@
     padding: .5em 2em;
     font-family: Avenir, Helvetica, Arial, sans-serif;
     animation-duration: .15s;
-    display: grid;
-}
\ No newline at end of file
+    display: flex;
```
Comment (maudetes) [inline theme/less/components/toast.less:132]: I think we have some issue with hover. See:
image

Comment (nicolaskempf57) [reply theme/less/components/toast.less:125]: Done ! This now extends the `.fr-btn` and use DSFR colors

**Review by maudetes — APPROVED** [review]
:sparkler:

**Pull request merged**